ros2/src/autorccar_costmap/include/costmap.h

```diff
@@ -0,0 +1,82 @@
+#ifndef COSTMAP_H
+#define COSTMAP_H
+
+#include <pcl/common/transforms.h>
+#include <pcl/filters/passthrough.h>
+#include <pcl/point_cloud.h>
+#include <pcl/point_types.h>
+
+#include <algorithm>
+#include <eigen3/Eigen/Dense>
+#include <fstream>
+#include <iostream>
+#include <string>
+#include <utility>
+#include <vector>
+
+struct CostmapInfo {
+    double resolution;    // map resolution [m/cell]
```

Comment:
strongly recommend)

```suggestion
    double resolution = 0.0;    // map resolution [m/cell]
```
안전한 코딩을 위해 항상 초기화하는걸 추천합니다 :+1: